<testng-results>
  <reporter-output>
  </reporter-output>
  <suite name="tac-tests" duration-ms="78" started-at="2010-07-08T18:44:12Z" finished-at="2010-07-08T18:44:12Z">
    <groups>
    </groups>
    <test name="com.mycompany.app.functest.FirstTest" duration-ms="78" started-at="2010-07-08T18:44:12Z" finished-at="2010-07-08T18:44:12Z">
      <class name="com.mycompany.app.functest.FirstTest">
        <test-method status="FAIL" signature="shouldPass()" name="shouldPass on instance null(com.mycompany.app.functest.FirstTest)" duration-ms="47" started-at="2010-07-08T18:44:12Z" finished-at="2010-07-08T18:44:12Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.mycompany.app.functest.FirstTest.shouldPass(FirstTest.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodHelper.invokeMethod(MethodHelper.java:643)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:559)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:723)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1027)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:137)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:121)
	at org.testng.TestRunner.runWorkers(TestRunner.java:953)
	at org.testng.TestRunner.privateRun(TestRunner.java:633)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:316)
	at org.testng.SuiteRunner.run(SuiteRunner.java:195)
	at org.testng.TestNG.createAndRunSuiteRunners(TestNG.java:903)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:872)
	at org.testng.TestNG.run(TestNG.java:780)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:75)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:127)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
      </class>
    </test>
  </suite>
</testng-results>
